auth/src/main/java/org/aerogear/auth/credentials/ICredential.java

```diff
@@ -4,4 +4,5 @@
  * Base interface for credential objects.
  */
 public interface ICredential {
+    public String getAccessToken();
```

Comment:
This is only about OIDC authentication, so it should be only inside the `OIDCCredential` object